pom.xml

```diff
@@ -197,7 +197,7 @@
     <maven.spotbugs.plugin.version>3.1.12.2</maven.spotbugs.plugin.version>
     <maven.pmd.plugin.version>3.12.0</maven.pmd.plugin.version>
     <pmd.version>6.17.0</pmd.version>
-    <maven.jacoco.plugin.version>0.8.4</maven.jacoco.plugin.version>
+    <maven.jacoco.plugin.version>0.8.5</maven.jacoco.plugin.version>
```

Comment:
Jacoco should be in its own commit, not in the same commit as jgit.

Reply:
Yes, by bad, it was a commit from phone. I will do it properly, but good that all CI pass, so I can merge it  after commit clean up

Reply:
I have split the commits into 2.